roles/dlrn_report/tasks/dlrn_report_results.yml

```diff
@@ -16,6 +16,14 @@
       -k -t {{ cifmw_dlrn_report_keytab }}
   when: cifmw_dlrn_report_kerberos_auth|bool
 
+- name: Set empty value for dlrnapi password
+  ansible.builtin.set_fact:
+    dlrnapi:
+      password: ""
+  when:
+    - dlrnapi is defined
```

Comment:
shouldn't dlrnapi.password be defined also when dlrnapi is not defined? Otherwise line 64 will fail